runtime/bcutil/cfreader.c

```diff
@@ -1128,12 +1128,20 @@ readPool(J9CfrClassFile* classfile, U_8* data, U_8* dataEnd, U_8* segment, U_8*
 			i++;
 			break;
 
+		case CFR_CONSTANT_Dynamic:
+			if (classfile->majorVersion < 55) {
+				errorCode = J9NLS_CFR_ERR_CP_ENTRY_INVALID_BEFORE_V51__ID;
```

Comment:
`J9NLS_CFR_ERR_CP_ENTRY_INVALID_BEFORE_V51__ID`  This should be v55 shouldn't it?

Reply:
@DanHeidinga you are right, I ll add a new message to `cfrerr.nls`, do I need to consider translations?

Reply:
Adding it to the end of the file is sufficient.  It'll get translated in the next translation pass